packages/server-core/src/networking/instance-provision/instance-provision.class.ts

```diff
@@ -483,6 +545,19 @@ export class InstanceProvision implements ServiceMethods<any> {
             (!config.kubernetes.enabled || (config.kubernetes.enabled && !isCleanup)) &&
             instance.currentUsers < location.maxUsersPerInstance
           ) {
+            if (roomCode) {
```

Comment:
should we be checking here that the roomCode provided actually corresponds to the instance? or is this handled elsewhere?

Reply:
Fixed